src/serialization/string_utils.cpp

```diff
@@ -702,7 +702,12 @@ bool word_match(const std::string& message, const std::string& word) {
 }
 
 bool wildcard_string_match(const std::string& str, const std::string& match) {
```

Comment:
This should take a string_view so substr doesn’t make new strings.

Reply:
I'm resolving this for now because it doesn't belong in this PR. (The glob commit was supposed to be a separate PR – it was pushed with this by mistake.)